index.html

```diff
@@ -21,51 +23,94 @@
-        <button>Submit</button>
+
+      <div class = "email-create">
+        <div>
+          <h2>New Schedule</h2>
+          <label>
+            <input name="email[to]" type='email' placeholder="To" />
+          </label>
+        </div>
+        <div>
+          <label>
+            <input name="email[subject]" type='text' placeholder="Subject" />
+          </label>
+        </div>
+        <div>
+          <label class = "email-body">
+            <input name="body-placeholder" type='text' placeholder="Body" />
+          </label>
+        </div>
+        <div>
+          <a href id = "schedule">Schedule Email</a>
+        </div>
+      </div>
+
+      <div class = "schedule-send">
+        <div>
+          <label>
+            <input type="text" id = "datepicker" name="email[send_first_at]" placeholder="Send at" />
+          </label>
+        </div>
+        <div>
+          <label>
+            <div class = "styled-select">
+            Repeat:
+            <select name="email[every_n_minutes]">
+              <option value="{{#daysToMinutes}}1{{/daysToMinutes}}">Daily</option>
+              <option value="{{#daysToMinutes}}2{{/daysToMinutes}}">2 Days</option>
+              <option value="{{#daysToMinutes}}3{{/daysToMinutes}}">3 Days</option>
+              <option value="{{#daysToMinutes}}4{{/daysToMinutes}}">4 Days</option>
+              <option value="{{#daysToMinutes}}5{{/daysToMinutes}}">5 Days</option>
+              <option value="{{#daysToMinutes}}6{{/daysToMinutes}}">6 Days</option>
+              <option value="{{#daysToMinutes}}7{{/daysToMinutes}}">Weekly</option>
+              <option value="{{#daysToMinutes}}30{{/daysToMinutes}}">Monthly</option>
+              <option value="{{#daysToMinutes}}365{{/daysToMinutes}}">Yearly</option>
+            </select>
+           </div>
+          </label>
+        </div>
+        <div>
+          <label>
+            <input name="email[times]" type='number' placeholder="Number of Times" />
+          </label>
+        </div>
+        <div>
+          <button class = "go-back">< Go Back</button>
+          <button>Submit</button>
+        </div>
+      </div>
+
       </form>
+
       <div class='emails-container'>
       </div>
       <div class='email-container'>
+        <h1>Subject</h1>
+          <ul>
+            <li><p>Sent To:</p></li>
+            <li><p>Sent At:</p></li>
+          </ul>
+          <p>
+          Body: Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed
+          do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+          ad minim veniam, quis nostrud exercitation ullamco laboris nisi
+          ut aliquip ex ea commodo consequat. Duis aute irure dolor in
+          reprehenderit in voluptate velit esse cillum dolore eu fugiat
+          nulla pariatur. Excepteur sint occaecat cupidatat non proident,
+          sunt in culpa qui officia deserunt mollit anim id est laborum.
+          </p>
+      </div>
+      <div>
```

Comment:
It looks like this is just an empty `div`, will you remove it just to keep things clean.